Makefile

```diff
@@ -108,7 +108,7 @@ cleanall: clean cleanall-deps
 DOCKER_CMD ?= "docker"
 
 build-base: install-deps protoc build-server build-gateway build-beat build-agent build-bootstrap build-monit
-build: build-base buildall-cli
+build: build-base build-cli
```

Comment:
@subfuzion you're right, it was to speedup the build a little. Let me rework that in another PR.